components/notebook-controller/go.mod

```diff
@@ -4,19 +4,11 @@ go 1.12
 
 require (
 	github.com/go-logr/logr v0.1.0
-	github.com/gogo/protobuf v1.2.1 // indirect
-	github.com/json-iterator/go v1.1.6 // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
-	github.com/onsi/ginkgo v1.8.0 // indirect
-	github.com/onsi/gomega v1.5.0 // indirect
+	github.com/kubeflow/kubeflow v0.7.0-rc.5.0.20191205174032-2da148904248
```

Comment:
Is this pinning to a specific version of kubeflow/kubeflow? Do we want to do that? 
Why is github.com/kubeflow/kubeflow listed go.mod? Isn't github.com/kubeflow/kubeflow the module we are building?